Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -945,17 +969,20 @@ void NetPlayDialog::OnHostInputAuthorityChanged(bool enabled)
 
     if (is_hosting)
     {
-      m_buffer_size_box->setEnabled(enable_buffer);
       m_buffer_label->setEnabled(enable_buffer);
```

Comment:
Removing this isn't the correct way, the logic should instead be amended to also check for auto buffer being enabled, otherwise the buffer spinbox will get enabled when it's not supposed to be.